wifiphisher/common/constants.py

```diff
@@ -1,6 +1,8 @@
 #!/usr/bin/env python2
 # -*- coding: utf-8 -*-
 # pylint: skip-file
+from __future__ import unicode_literals
+from builtins import range
```

Comment:
@sophron Thanks for catching that. I included the `future` library in the `setup.py` but for some reason it still fails. Any idea why?

Reply:
It's the `builtins` library that is missing.

In overall, I'm against adding a new dependency for the sake of Python v3 compliancy.

Reply:
That dependency will only be temporary. After the code is `2/3` compatible we will remove the `future` library and get rid of the boilerplate. This method allows us to still work on the project as opposed to stopping all development and change the code base to `python 3`. I opened #426 January of 2017. It is now June of 2018 and we have not converted a single line of code. With only a year and half left on `python 2` life support I think it is time to move on to `python 3`.

Reply:
I think the most proper way is to have a specific branch of Wifiphisher that runs on Python 3. When the time comes, we can switch our master to that branch.

But for now, master doesn't need to include these.

Reply:
@sophron I'm fine with this but what happens if we add a feature or fix a bug in `master`? It will need to be merged in both `master` and `python3` branch and there might be some differences there.

Reply:
It will be merged in `master` but periodically the `python3` branch should be synced.